setup.py

```diff
@@ -128,8 +128,22 @@ def get_extensions():
             if liberfa_versions:
                 print('Configure liberfa ("configure.ac" scan)')
                 lines = []
-                for name, value in get_liberfa_versions():
-                    lines.append(f'#define {name} "{value}"')
```

Comment:
I think this can be done more simply as,
```
if isinstance(value, str):
    # strings should be escaped
     value = f'"{value}"'
lines.append(f'#define {name} {value}')
```
Or, just accounting for the fact that `repr` gives the wrong type of quote,
```
lines.append(f'#define {name} {value!r}'.replace("'", '"'))
```

Reply:
neat. I just pushed your one-liner. Thanks !